internal/cmd/destinations.go

```diff
@@ -90,6 +90,11 @@ func newDestinationsRemoveCmd(cli *CLI) *cobra.Command {
 			logging.S.Debugf("call server: list destinations named %q", name)
 			destinations, err := client.ListDestinations(api.ListDestinationsRequest{Name: name})
 			if err != nil {
+				if api.ErrorStatusCode(err) == 403 {
+					return Error{
+						Message: "Cannot disconnect destination: missing privileges for ListDestinations",
+					}
+				}
```

Comment:
if you're going to replace the existing error like this, you need to debug log the original error.

Reply:
Isn't the original error always just `Forbidden` , since we hide everything in `sendAPIError` ?

Reply:
done